src/help.cpp

```diff
@@ -21,8 +21,7 @@ std::vector<std::string> hints;
 std::vector<std::string> text_intro;
 } //namespace
 
-// @todo add required category id as an input parameter
-void load_help()
+void load_help( const std::string required_category )
 {
     std::string path = FILENAMES["help_notes"].c_str();
```

Comment:
`FILENAMES` is a map of `std::string`s. You can assign `path` without constructing the string from `*char`
```c++
const std::string path = FILENAMES["help_notes"]; 
// or even
const auto path = FILENAMES["help_notes"]; 
```